styles/vim-mode.less

```diff
@@ -20,6 +20,15 @@
   opacity: 0.5;
 }
 
+.low-block-cursor(@visibility: visible) {
+  border: none;
+  border-bottom: 10px solid @syntax-cursor-color;
```

Comment:
may I suggest .6em rather than 10px here? When I pasted my custom styles, I didn't remember to change to relative units.